modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -19,9 +19,9 @@ If you manually changed your existing installation, such as, changing the target
 The {pipelines-title} Operator now provides the option to choose the components that you want to install by specifying profiles as part of the `TektonConfig` CR. The `TektonConfig` CR is automatically installed when the Operator is installed.
 The supported profiles are:
 
-* Basic: This installs only Tekton pipelines.
-* Default: This installs Tekton pipeline and Tekton triggers.
-* All: This is the default profile used when the `TektonConfig` CR is installed. This profile installs all of the Tekton components: Tekton Pipelines, Tekton Triggers, Tekton Addons(which include `ClusterTasks`, `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStart` and `ConsoleYAMLSample` resources).
+* Lite: This installs only Tekton Pipelines.
+* Basic: This installs Tekton Pipelines and Tekton Triggers.
+* All: This installs all of the Tekton components: Tekton Pipelines, Tekton Triggers, Tekton Addons(which include `ClusterTasks`, `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStart` and `ConsoleYAMLSample` resources).
```

Comment:
The new phrasing does not make it clear which is now the default installation.

Reply:
Just now Nikhil confirmed that 'All' is still the default profile in 1.5. I will update the documentation. Thanks.